pay.js

```diff
@@ -0,0 +1,45 @@
+const IlpPacket = require('ilp-packet')
+const Plugin = require('ilp-plugin-xrp-escrow')
+const uuid = require('uuid/v4')
+function base64url (buf) { return buf.toString('base64').replace(/\+/g, '-').replace(/\//g, '_').replace(/=+$/, '') }
+
+const destinationAddress = process.argv[2]
+const destinationAmount = process.argv[3]
+const condition = process.argv[4]
+
+const plugin = new Plugin({
+  secret: 'sndb5JDdyWiHZia9zv44zSr2itRy1',
+  account: 'rGtqDAJNTDMLaNNfq1RVYgPT8onFMj19Aj',
+  server: 'wss://s.altnet.rippletest.net:51233',
+  prefix: 'test.crypto.xrp.'
+})
+
+function sendTransfer (obj) {
```

Comment:
It's not strictly necessary but I think it would be nice to add a bunch of comments in all of the source files detailing what's going on